plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -4137,6 +4137,10 @@ function wc_get_cart_undo_url( $cart_item_key ) {
  * @since 3.5.0
  */
 function woocommerce_output_all_notices() {
+	if ( ! function_exists( 'wc_print_notices' ) ) {
+		return;
```

Comment:
```suggestion
		include_once WC_ABSPATH . 'includes/wc-notice-functions.php';
```

What if we just loaded the file here when necessary?

Reply:
Nevermind, I see that the function still wouldn't do anything since it requires that the session object is initialized first. Returning as you put here is probably the best option.

Might be worth adding a note in the doc block for this function that it is only intended for front end use. And maybe even a `_doing_it_wrong()`?

Reply:
Yeah, I thought about including the file, but decided against it because it seems like this is an indication that something's off and we shouldn't just bring everything in (and there's also sessions and stuff as you mentioned).

> Might be worth adding a note in the doc block for this function that it is only intended for front end use. And maybe even a _doing_it_wrong()?

Given this isn't really interacting with anything (i.e. it's not adding to the cart or something like that) I didn't think it was worth it, but guess better safe than sorry.
I've added one in commit e74d19ddf04480238e0209d4d567657b07f79546. Let me know what you think.

Reply:
👍 That looks good. I was wondering if the message string should be translatable, but it looks like we are not consistent one way or the other in the codebase, and it would only be seen when WP_DEBUG is turned on anyway, so I think it's fine to not do it here.